Dockerfile

```diff
@@ -162,10 +165,10 @@ CMD ["sh", "-c", "${DEV_START}"]
 # Stage 3: Extract a minimal image from the build                 #
 ###################################################################
 
-FROM helsinkitest/node:16-slim  AS runner
+FROM gcr.io/distroless/nodejs16-debian11 AS runner
 
 # Use non-root user
-USER appuser
+USER nobody
```

Comment:
Decision not to use the user would compromise the security of the image: https://www.geeksforgeeks.org/docker-user-instruction/

Reply:
Here's what lead me to this: https://github.com/GoogleContainerTools/distroless/issues/332 . So my understanding was that nobody on these distroless containers is just a non-root user. But that said, it wouldn't hurt to just add a new one; there's simple steps on that same issue how to do that..

Reply:
Ok! Maybe extra comment about this with link might help understanding for anyone who are reading this file later